### DISCUSSION TITLE
Tailwind CSS와 전통적인 CSS 프레임워크의 성능 비교

### DISCUSSION THREAD

**Discussion by ji006** [100-hours-a-week/discussions#37] [Deepdive]
Heartfolio는 tailwind css를 사용하였습니다. 이에 대한 성능을 비교해보겠습니다.
* Tailwind CSS와 Bootstrap, Foundation 등 전통적인 CSS 프레임워크와의 성능을 비교
* 각 프레임워크의 CSS 파일 크기, 로딩 시간, 유지보수 용이성, 유연성 등을 주요 지표로 삼아 분석
* 개발 속도와 코드 재사용성 측면에서 두 접근 방식의 차이점을 탐구
---
# css란?
#### 1. External Stylesheets
* 분리된 디자인 파일로 관리
* 적용하고 싶은 html 파일 내부 head 태그에 link 태그를 통해 정의 필요

#### 2. Inline Styles
* 관련 요소에 스타일 속성을 추가

#### 3. Internal Stylesheets
* html 파일 내부 head 태그에 style 태그 안에 정의함

#### 6. Preprocessors (Sass)

.

#### 5. Tailwind CSS

**Comment by startupcodekr** [comment]
"대량의 실시간 주가 데이터를 웹소켓을 통해 수신할 때 발생하는 성능 이슈를 해결하기 위한 최적화 전략"

**Reply by ji006** [reply]
대량이 아니라 바꾸겠습니다.

**Comment by startupcodekr** [comment]
실시간 주식 데이터 반영 지연 문제 해결 및 UI 개선 방안: 프론트엔드 성능 모니터링을 통한 실시간 데이터 처리 최적화 전략

**Reply by ji006** [reply]
수정했습니다!
